src/views/OrderDetail.vue

```diff
@@ -3,7 +3,7 @@
     <ion-header :translucent="true">
       <ion-toolbar>
         <ion-back-button slot="start" default-href="/purchase-order" />
-        <ion-title>{{ orderId }}</ion-title>
+        <ion-title>{{ $t("Review purchase order") }}</ion-title>
```

Comment:
``<ion-title>{{ $t("Review purchase order") }}</ion-title>
``
Add this static text in translation file & export ion-title

Reply:
Added.